introduction.md

```diff
@@ -0,0 +1 @@
+hi there!, this is my succinct introduction to the dojo team! :).
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.